pkg/agent/flowexporter/connections/connections.go

```diff
@@ -28,42 +27,64 @@ import (
 var _ ConnectionStore = new(connectionStore)
 
 type ConnectionStore interface {
-	Run(stopCh <-chan struct{})
+	Run(stopCh <-chan struct{}, pollDone chan bool)
```

Comment:
if this is used as a signal, I believe a more common pattern in go is to use an empty struct?

Reply:
Since I picked an option of multiple signals, I kept the channel with data. Empty struct seems to be ideal when sending signal with close(channel). Correct me if my understanding is wrong.